mon-pix/app/components/routes/campaigns/assessment/skill-review.js

```diff
@@ -174,12 +176,16 @@ export default class SkillReview extends Component {
     return this.args.model.campaignParticipationResult.canImprove && !this.isShareButtonClicked;
   }
 
+  get showBadges() {
+    return this.showCertifiableBadges || this.showNotCertifiableBadges;
+  }
+
   get showHeavyBlock() {
     return (
-      this.showCertifiableBadges ||
-      this.showNotCertifiableBadges ||
+      this.showBadges ||
       this.showImproveButton ||
-      !this.args.model.campaign.isForAbsoluteNovice
+      !this.args.model.campaign.isForAbsoluteNovice ||
+      (this.showBadges && this.isAutonomousCourse)
```

Comment:
```suggestion
      !this.args.model.campaign.isForAbsoluteNovice ||
      this.isAutonomousCourse
```

À ce stade là de la condition, `this.showBadges` est forcément `false`.
Sinon on aurait validé la première condition.

Est-ce que le fait d'être un parcours autonome ne devrait pas être une condition valable pour que `showBadges` soit `true` ?

Reply:
Non bien vu, c'est un reliquat :)